packages/jest-mock-scheduler/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "jest-mock-scheduler",
+  "version": "0.1.0",
+  "description": "Jest matchers and utilities for testing the scheduler package.",
+  "main": "index.js",
+  "repository": "facebook/react",
+  "keywords": [
+    "jest",
+    "scheduler"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/facebook/react/issues"
+  },
+  "homepage": "https://reactjs.org/",
+  "peerDependencies": {
+    "jest": "^23.0.1",
+    "scheduler": "^0.11.0"
+  },
+  "files": [
+    "LICENSE",
+    "README.md",
+    "build-info.json",
```

Comment:
What is this?

Reply:
`build-info` is metadata managed and used by our release scripts.

Reply:
What's in it? Sorry if I missed it somewhere. Do we include it in all packages?

Reply:
Just some metadata about this build, e.g.
https://unpkg.com/react@canary/build-info.json